apps/mosaico/src/Mosaico.purs

```diff
@@ -193,7 +194,11 @@ mosaicoComponent initialValues props = React.do
         | otherwise ->
           Aff.launchAff_ do
             staticPage <- fetchStaticPage page
-            liftEffect $ setState _  { staticPage = Just staticPage }
+            liftEffect $ setState _  { staticPage = Just staticPage }  
+            case staticPage of
+              StaticPageResponse r
+                | Just p <- r.pageScript -> liftEffect $ evalExternalScripts [ScriptTag $ "<script>" <> p <> "</script>"]
```

Comment:
We can't do that as the function expects full script tags as input (which is what we get from Lettera). But, we do actually have `ExternalScript` (https://github.com/KSF-Media/affresco/blob/master/packages/components/src/Lettera/Models.purs#L119) already. We could use that here instead of `ScriptTag` and create a `mkExternalScript`, or something, that takes in a `String` (the script) and wraps it in `<script>x</script>`.